gcc/rust/lint/rust-lint-marklive.h

```diff
@@ -266,10 +266,9 @@ class MarkLive : public MarkLiveBase
       tyctx (Resolver::TypeCheckContext::get ()){};
 
   void mark_hir_id (HirId);
-  bool visit_path_segment (HIR::PathExprSegment);
```

Comment:
I think keep this bool visit_path_segment for now

Reply:
May I ask why shoud I keep this bool for now?

Reply:
it may present other options for more error handling later on.